core/scripts/server.ts

```diff
@@ -186,6 +202,15 @@ app.get('*', (req, res, next) => {
           tagsArray
         ).catch(errorHandler)
       }
+
+      const hookResponse2 = serverHooksExecutors.afterOutputRendered({
+        req,
+        res,
+        context,
+        output,
+        isProd
+      })
+
       res.end(output)
```

Comment:
if `afterOutputRenderedResponse` !== null it should be then used instead of `output`

Reply:
I have used this logic here: 

```typescript
      if (typeof afterOutputRenderedResponse.output === 'string') {
        res.end(afterOutputRenderedResponse.output)
      } else if (typeof afterOutputRenderedResponse === 'string') {
        res.end(afterOutputRenderedResponse)
      } else {
        res.end(output)
      }
```

As this should be more what we want I think.